src/evaluate5.jl

```diff
@@ -23,6 +23,7 @@ for 5-card hands _only_.
 function evaluate5 end
 
 evaluate5(cards::Card...) = evaluate5(cards)
+evaluate5(cards::SubArray) = evaluate5(Tuple(cards))
```

Comment:
I'm realizing that this may be resulting in allocations. Going to try a slightly different internal interface.